help/marking-messages-as-read.md

```diff
@@ -9,14 +9,20 @@ configure whether messages are marked as read automatically when you scroll.
 
 ## Configure whether messages are automatically marked as read
 
-You may want to configure whether or not the Zulip mobile app will automatically
-mark messages as read. For example, it can be useful to set the app to never
-mark messages as read if you often want to follow up on messages when you
+You may want to configure whether or not the Zulip app will automatically
+mark messages as read. For example, it can be useful to set the mobile app to
+never mark messages as read if you often want to follow up on messages when you
 are at your computer. You will still be able to
 [manually](#mark-all-messages-as-read) mark messages as read.
 
 {start_tabs}
 
+{tab|desktop-web}
+
+{settings_tab|display-settings}
+
+1. Under **Advanced**, select **Do not mark messages as read on scroll**.
```

Comment:
Since this section is about configuring whether messages are automatically marked as read or not, we should use "toggle" instead of "select".
```suggestion
1. Under **Advanced**, toggle **Do not mark messages as read on scroll**.
```

Reply:
Thank you for the review! I think I'll hold this suggestion off for now as there is a disscussion where we may add an additional option to this.

Reply:
Yeah, let's call this "select", since it's a dropdown and we do plan to add more options.